README.md

```diff
@@ -126,32 +115,58 @@ Workmanager.registerOneOffTask(
 ```
 
 ### BackoffPolicy
-When a job should fail this specifies the waiting strategy it should use.  
+Indicates the waiting strategy upon task failure.  
 The default is `BackoffPolicy.exponential`.    
-You can also specify the delay.  
+You can also specify the delay. 
 
 ```
 Workmanager.registerOneOffTask("1", "simpleTask", backoffPolicy: BackoffPolicy.exponential, backoffPolicyDelay: Duration(seconds: 10));
 ```
 
 ## Cancellation
 
-You can cancel jobs in different ways.  
-### By Tag
+A taks can be cancelled in different ways :  
+- ##### by Tag
 
-If you have provided job with a tag, you can cancel it that way too.  
+Cancels the task that was previously registered using this **Tag**, if any.  
 
 ```
 Workmanager.cancelByTag("tag");
 ```
 
-### By Unique Name
+- ##### by Unique Name
 ```
-Workmanager.cancelByUniqueName("tag");
+Workmanager.cancelByUniqueName("<MyTask>");
 ```
 
-### All
+- ##### cancel all registered tasks
 
 ```
 Workmanager.cancelAll();
 ```
+
+## iOS usage
+
+Background taks on iOS are very different. Before anything, make sure you've added the following key to your project's `Info.plist :
+```
+<key>UIBackgroundModes</key>
+  <array>
+    <string>fetch</string>
+  </array>
+</key>
+```
+
+#### Set the application's minimumBackgroundFetchInterval
+
+Set your desired *minimumBackgroundFetchInterval* in your app's delegate's `didFinishLaunchingWithOptions` :
+
+`UIApplication.shared.setMinimumBackgroundFetchInterval(TimeInterval(60 * 15)) // E.g. max every 15 minutes`
+> Note : this time interval is a minimum ; there's no guarantee about how ofteh this will be called. 
+
+#### Adding the plugin as an AppDelegate
+
+This is done automatically by the SwiftWorkManagerPlugin at registration time, by calling [addApplicationDelegate](https://api.flutter.dev/objcdoc/Protocols/FlutterPluginRegistrar.html#/c:objc(pl)FlutterPluginRegistrar(im)addApplicationDelegate:) on the Flutter registrar.
```

Comment:
Is it necessary to inform the user of this automatic action? 
I think it can only be confusing, no?

Reply:
I believe it's added value ; as a user of this plugin, I'd have a hard time figuring out how the plugin intercepts the native `performFetch`delegate call (except some Method Swizzling/dark magic thing).

Reply:
But you should not care as the implementer. If you care you check the code.
I try to keep the README as terse as possible